lib/PuppeteerSharp/ExecutionContext.cs

```diff
@@ -173,8 +174,8 @@ internal async Task<JSHandle> EvaluateFunctionHandleAsync(string script, params
             }
         }
 
-        internal JSHandle CreateJSHandle(dynamic remoteObject)
-            => (remoteObject.subtype == "node" && Frame != null)
+        internal JSHandle CreateJSHandle(JToken remoteObject)
```

Comment:
do we want to create a class for this?

Reply:
fair enough

Reply:
I will implement a class @Meir017 the shape will be a composition of that ;)